session/bootstrap.go

```diff
@@ -1597,12 +1588,7 @@ func upgradeToVer19(s Session, ver int64) {
 	doReentrantDDL(s, "ALTER TABLE mysql.columns_priv MODIFY User CHAR(32)")
 }
 
-func upgradeToVer20(s Session, ver int64) {
-	if ver >= version20 {
-		return
-	}
-	doReentrantDDL(s, CreateStatsFeedbackTable)
-}
+func upgradeToVer20(_ Session, _ int64) {}
```

Comment:
Do we need to keep it or clean up the old table here?

Reply:
I suggest that we'd better keep all system tables unchanged for safety.

Reply:
Reverted. Thanks!